tap_klaviyo/__init__.py

```diff
@@ -134,7 +136,8 @@ def get_available_metrics(api_key):
                         stream=EVENT_MAPPINGS[metric['name']],
                         tap_stream_id=metric['id'],
                         key_properties="id",
-                        replication_method='INCREMENTAL'
+                        replication_method='INCREMENTAL',
+                        replication_keys=["since"]
```

Comment:
Tap writing bookmark value in the `since` key but it is not an actual field in the API response. API response contains the `timestamp` field and that value is being written in the `since` field. It is an existing behavior of the tap.